lib/turn_utils.ex

```diff
@@ -0,0 +1,105 @@
+defmodule Membrane.TURN.Utils do
+  @moduledoc false
+
+  @numbers_and_letters Enum.concat([?0..?9, ?a..?z, ?A..?Z])
+
+  @spec generate_credentials(binary(), binary()) :: {binary(), binary()}
+  def generate_credentials(name, secret) do
+    duration =
```

Comment:
Would it be possible to reference here source of formula for generating credentials?

Reply:
In this case, algorithm for generating password for specific user is not specified in any RFC and may differ between implementations